### PR TITLE
Changing self.logger to logger

### PR DESCRIPTION
**Pull request by YoniItzhak**

### CODE REVIEW

**Comment by YoniItzhak** [comment]
The different components use two different loggers: `self.logger` defined [here](https://github.com/cloudify-cosmo/cloudify-manager-install/blob/5e8b6349b86d0acf2c5d2db9731bbcd49d832b4f/cfy_manager/components/base_component/base_component.py#L26) and `logger` that is defined in every component, like [here](https://github.com/cloudify-cosmo/cloudify-manager-install/blob/5e8b6349b86d0acf2c5d2db9731bbcd49d832b4f/cfy_manager/components/postgresql_server/postgresql_server.py#L144). 
This causes the `cfy_manager` commands to log different names for the same component. E.g. `POSTGRESQLSERVER` and `POSTGRESQL_SERVER`.

This PR fixes the issue by changing `self.logger` to logger in the relevant components.

The question is why do we even need two different attributes for logger.

File: cfy_manager/components/nginx/nginx.py [hunk 1]
```diff
@@ -196,7 +196,7 @@ def _replace_external_certs(self):
         self._handle_external_cert(replacing_ca=replacing_ca)
 
     def log_replacing_certificates(self, certs_type):
-        self.logger.info('Replacing %s on nginx component', certs_type)
+        logger.info('Replacing %s on nginx component', certs_type)
```
Comment (tehasdf) [inline cfy_manager/components/nginx/nginx.py:199]: if you want more -lines then just inline these stupid methods

we _really_ don't need a layer for writing a single log message

File: cfy_manager/components/base_component/base_component.py [hunk 2]
```diff
@@ -85,26 +83,6 @@ def validate_dependencies(self):
                 'All prerequisites for {class_name} are met'.format(
                     class_name=self.__class__.__name__))
 
-    def use_supplied_certificates(self,
```
Comment (tehasdf) [inline cfy_manager/components/base_component/base_component.py:88]: awesome

**Review by geokala — COMMENTED** [review]
I'd like to see example output from this if you have some to hand?

**Review by tehasdf — APPROVED** [review]

**Comment by tehasdf** [comment]
@geokala you can see the output in the CI builds, the "waiting for container to start" part of each job in install_cluster & install_manager has it (with verbose, too)

**Comment by geokala** [comment]
Yeah, that would've been smart.

OK, looks about the same as before to me, which is unsurprising given the issue it was targeting. Will approve.

**Review by geokala — APPROVED** [review]
I agree with @tehasdf about removing the unnecessary function, but that's not critical, so approved.

**Pull request merged**